grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -130,6 +130,13 @@ class NewConsentGroupController extends AuthenticatedController {
         render([message: 'Consent Group was updated'] as JSON)
     }
 
+    def deleteNoConsentReason() {
+        Issue issue = Issue.findByProjectKey(params.consentKey)
+        issueService.removeNoConsentAnswer(issue)
+        response.status = 200
+        render([message: 'Deleted No Consent Form reason'] as JSON)
```

Comment:
APIs, in general, should return the thing that is modified. In this case, it should be returning the modified issue.